### PR TITLE
Tribute page modification

### PR DESCRIPTION
**Pull request by sheldonwafula**
In this pull request:
-I have created a tribute page with HTML.
-I have added CSS for styling.
-I have added a descriptive readme file.
-I have fixed all linter errors.

### CODE REVIEW

**Comment by sheldonwafula** [comment]
1. The project works properly.
2. No linters errors.
3. Correct flow used.
4. well-documented repository.

**Comment by sheldonwafula** [comment]
I think it would be better to align the button properly. Reducing the left margin to align with the rest of the content.

**Comment by sheldonwafula** [comment]
I think it would be better to align the heading at the center  <h1 id="title">Nelson Mandela</h1>.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <main id="main">
+        <h1 id="title">Nelson Mandela</h1>
+        <div id="img-div">
+            <img class="image" src="image/pexels-minenhle-shelembe-5985865.jpg" alt="Nelson Mandela">
+            <figcaption id="img-caption">Picture of Nelson Mandela</figcaption>
+        </div>
+        <div class="tribute-info" >
+            <p>About Nelson madela</p>
+            <a class="info" href="https://en.wikipedia.org/wiki/Nelson_Mandela" target="_blank">Read More about our hero Nelson Mandela</a>
+        </div>
```
Comment (didierganthier) [inline index.html:20]: For example, this is an inline comment 🙂 from lines `17` to `20`, it appears below the review. Also make sure that when you add one, you click on `Start a review` and not `Add single comment`. The difference? 🤔 If you click on `Add single comment`, the comment will appear before the review and seem unrelated to it, but if you click on `Start a review`, it only gets posted once you submit the review and appears below it. I hope this helps, let me know if you have any questions ❓ 😉

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @sheldonwafula ,

🎉 Good job so far! You're making significant progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Used bullet points ✔️ 

## Required Changes 🛠
- [ ] Great job on the comments you made 👍🏽 but in order to improve the review and make it look better, you should consider using the "Review changes" feature from GitHub. It makes it easier to "Comment", "Approve" or "Request Changes". In order to do that you can follow these steps:
1. Files changed 
Screenshot 2023-01-23 at 07 57 28
2. Review changes
Screenshot 2023-01-23 at 07 57 47
3. Write your review (in markdown format)
Screenshot 2023-01-23 at 07 58 23
4. Comment, Approve or Request changes
Screenshot 2023-01-23 at 07 59 38
5. Submit review
Screenshot 2023-01-23 at 08 00 02

- [ ] To make this even better 😉 make sure you also use inline comments as requested for this project 💭 inline comments help the student understand exactly what part of the code you're commenting about. See below for an example 👇🏽 

- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

**Review by sheldonwafula — COMMENTED** [review]
- I think it would be better to align the heading at the center.
 Screenshot 2023-01-24 115328
- I think it would be better to align the button properly. Reducing the left margin to align with the rest of the content.
 Screenshot 2023-01-24 115842

**Review by sheldonwafula — COMMENTED** [review]
Please add .LICENSE to your README file. Your readme appears to have no license details.
Screenshot 2023-01-24 121524

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @sheldonwafula     👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
I notice you did make efforts as regards suggestions made by the previous reviewer to use inline comments, However, you still need to work on that to move to the next project. 💯 



Highlights
✅All linters are checked
✅ Descriptive commit messages 


Required Changes ♻️

- [ ] What you appeared to have done, is take a screenshot of changes you suggested based on the review task however, the guide to make incline comments is different. [This link ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) has a step-by-step guide on how to do that. please kindly revisit it to see the workflow 👍 
## Vidoe link for inline comment sample 
[Video link](https://www.loom.com/share/e847c46aa4814534b23b3a8e4dd60d6d)

_check the comment section for a detailed review_

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.


Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Shedrack-Sunday** [comment]
- [ ] You're not meant to use a snapshot. I've taken a screenshot of the process phase and attached links and video path to help you see how its done in my review below 

## Expected process :
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page Built by Sheldon and Muda</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <main id="main">
+        <h1 id="title">Nelson Mandela</h1>
```
Comment (sheldonwafula) [inline index.html:12]: I think it would be better to align the heading at the center,to make your page look better.

File: README.md [hunk 3]
```diff
@@ -1 +1,180 @@
...
+
+I would like to thank my long time friend Mudasir for his
+endless support from the beginning of this project upto the end.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (sheldonwafula) [inline README.md:177]: Please add .LICENSE to your README file. Your readme appears to have no license details.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        <div id="img-div">
+            <img class="image" src="image/pexels-minenhle-shelembe-5985865.jpg" alt="Nelson Mandela">
+            <figcaption id="img-caption">Picture of Nelson Mandela</figcaption>
+        </div>
+        <div class="tribute-info" >
+            <p>About Nelson madela</p>
+            <a class="info" href="https://en.wikipedia.org/wiki/Nelson_Mandela" target="_blank">Read More about our hero Nelson Mandela</a>
```
Comment (sheldonwafula) [inline index.html:19]: I think it would be better to align the button properly. Reducing the left margin to align with the rest of the content.

**Review by sheldonwafula — COMMENTED** [review]
CHANGES REQUIRED.
Please make the changes and  If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Review by IngDavidTM — APPROVED** [review]
Hi <STUDENT_NAME> 🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The review uses inline comments ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**